content/news/playtest-20150102.md

```diff
@@ -0,0 +1,30 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement the unique features of Dune 2000 that distinguish it from the other mods.
+
+
+The playtest-20150102 build is available from the [download](/download/) page. We handed over packaging in regards of RPM for Fedora and openSUSE as well as Arch Linux completely to the Linux community as they produce better results than our build servers. Note to packagers: we switched dependency management to [NuGet](http://www.nuget.org/).
+
+These playtest builds provide a snapshot of our current development for feedback by our testers. You can help shape the future of OpenRA by testing these builds and [reporting bugs](http://bugs.openra.net) or simply letting the [community](/community/) know what you think.
+
+Some highlights in this development build are:
+
+   - Added an often requested lobby option for disabling the "Short Game" mode.
+   - Support hardware cursors for smoother mouse input using SDL 2.
+   - Added new missions GDI 03, Soviet 01 and Survival 01 as well as difficulties for Allies 03a.
+   - Sub-factions (Countries) for Red Alert are now available.
+   - Carryalls and Sandworms have been introduced in Dune 2000.
+
+See the [changelog](http://changelog.openra.net) for more details on the playtest changes and bugfixes.
+
+<div style="text-align:center" markdown="1">
+![Sandworms and Carryalls](/images/news/20150102-sandworms-carryalls.png)
```

Comment:
The carryall is lost in the visual noise of the base, and doesn't appear to be carrying a harvester.  See the wiki page for some suggestions on creating these screenshots.